ragaai_catalyst/tracers/agentic_tracing/upload/trace_uploader.py

```
@@ -349,6 +427,7 @@ def do_sync_processing():
                 dataset_name=dataset_name,
                 user_details=user_details,
                 base_url=base_url,
+                tracer_type = tracer_type,
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)